package.json

```diff
@@ -17,6 +17,7 @@
     "eslint": "6.7.2",
     "eslint-config-airbnb-base": "14.0.0",
     "eslint-plugin-import": "2.18.2",
+    "fomantic-ui": "2.8.3",
```

Comment:
Still need to move this to `dependencies`.

Reply:
Now I see where this is going: `depedencies` for building `gitea` and the `devDependencies` for ci.

`webpack` and all the plugins/loader should be moved into `dependencies` then - subject to another PR of course.

Reply:
Ideally we'd use `npm install --only=prod` for the build and `npm install --only=dev` for tests but that will require a few more changes in the way tests run, yeah. I'd like to reserve `dependencies` to actual JS modules loaded at runtime in the browser.